.gitignore

```diff
@@ -53,3 +53,4 @@ proxy_temp
 fastcgi_temp
 client_body_temp
 utils/lj-releng
+sftp-config.json
```

Comment:
where is this file come from?

Reply:
this is a sublime plugin config file, I use to accidentally commit it to repo, I think maybe there is another developer have the same issue, so  I add it to the ignore file